thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/detector/onboard-notification-email-template.ftl

```diff
@@ -67,12 +67,28 @@
               </tr>
               <tr>
                 <td colspan="1" style="color: #606060; font-size: 14px; line-height: 20px;">
-                  Mailing list
+                  Mailing list (to)

```

Comment:
minor: I'd call this "Recipients", because "Mailing list" suggests a single email list to be entered here